include/mbedtls/ecp.h

```diff
@@ -196,6 +196,9 @@ mbedtls_ecp_point;
  * cardinality is denoted by \p N. Our code requires that \p N is an
  * odd prime as mbedtls_ecp_mul() requires an odd number, and
  * mbedtls_ecdsa_sign() requires that it is prime for blinding purposes.
+ * The default implementation sets \p A to NULL rather than the authentic value
```

Comment:
`A` isn't a pointer. This should read something like “sets A to 0 (with A.p == NULL) rather than …”

Reply:
We should be careful about the wording here: setting A to 0 with `A.p != NULL` actually means zero. I think using parentheses here is dangerous, as this could be interpreted as optional while it's actually essential.